src/Media/Image.php

```diff
@@ -45,12 +45,12 @@ final class Image implements Serializable
     private $language;
 
     public function __construct(
-        UUID $id,
-        MIMEType $mimeType,
-        Description $description,
+        UUID      $id,
```

Comment:
This has some automatic formatting done by PHPStorm, better to avoid this.

You can also disable this
<img width="1399" alt="Screenshot 2021-12-03 at 11 22 52" src="https://user-images.githubusercontent.com/1322949/151940607-b63e1b98-5c37-438b-a673-8f92ddd6424e.png">